packages/client/webapp/src/components/Header.vue

```diff
@@ -1,33 +1,71 @@
 <script setup lang="ts">
-import { computed } from 'vue';
+import Button from '@/components/Button.vue';
 import IconLogin16 from '@carbon/icons-vue/es/login/16';
+import IconSearchLocate16 from '@carbon/icons-vue/es/search--locate/16';
+import { useRouter } from 'vue-router';
 import { useAuthStore } from '../stores/auth';
 
 const auth = useAuthStore();
-const isAuthenticated = computed(() => auth.isAuthenticated);
+const router = useRouter();
 
 const login = () => window.location.assign('http://localhost:8078');
+const homepage = () => router.push('/');
+
+const projectName = 'Name of the project that can be long for clarity and precision';
 </script>
 
 <template>
-	<nav>
-		<!-- Use relative path instead of alias for now as Component Tests seem to have an issue -->
+	<header>
 		<img src="@assets/images/logo.png" height="32" width="128" alt="logo" />
-		{{ auth.name }}
-		<Button v-if="!isAuthenticated" @click="login">Login <IconLogin16 /></Button>
-	</nav>
+		<p>
+			<a @click="homepage">Projects</a>
+			<span>{{ projectName }}</span>
+		</p>
+
+		<aside>
+			<Button class="dataExplorer" @click="router.push('/explorer')"><IconSearchLocate16 /></Button>
```

Comment:
Not sure if we established this convention team-wide, how do folks feel about consistently using `kebab-case` CSS classes?